app/src/main/java/com/shagalalab/constitution/part/PartFragment.kt

```diff
@@ -32,6 +38,10 @@ class ListFragment(private var lang: Int) : Fragment(R.layout.fragment_list), It
     }
 
     override fun onItemClick(model: PartModel) {
-        Toast.makeText(context, model.title, Toast.LENGTH_SHORT).show()
+        if (model.description.isEmpty())
+            Toast.makeText(context, model.title, Toast.LENGTH_SHORT).show()
+        else {
+            (activity as MainActivity).changeFragment(ChapterFragment(model.id), TAG)
+        }
```

Comment:
This logic is wrong, it works with `Preamble`, but `Part six` doesn't have any chapters too, but it is opening empty chapters list. Read carefully the description I wrote, there I mentioned how the logic should work. Also you are missing curly braces for if statement.